source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -205,3 +205,5 @@ inserts a new document with the specified fields and values.
 You cannot modify the ``_id`` field of a document nor change a field to
 a value that violates a unique index constraint. See the MongoDB Server manual
 for more information on :manual:`unique indexes </core/index-unique/>`.
+
+.. include:: /includes/undefined-values-note.rst
```

Comment:
I'm undecided on whether this was the right approach, as opposed to making different content for these pages based on the type of write op that they explain. I also couldn't find the right place to put this material, as it didn't seem appropriate to make a whole subsection for it on each page.

Reply:
We have the UTF-8 Validation docs as a top level page but this and that UTF page are both BSON controls. Maybe there's a merging or nesting that would be sensical given that. 

I foresee a near future where we may be adding more coverage of the BSON controls that the driver offers, we just have to [make the options more easily understood first](https://jira.mongodb.org/browse/NODE-4780).

Just a suggestion, of course!